src/PDE/Limiter.cpp

```diff
@@ -170,13 +170,26 @@ Superbee_P1( const std::vector< int >& esuel,
 // *****************************************************************************
 {
   const auto rdof = inciter::g_inputdeck.get< tag::discr, tag::rdof >();
+  const auto ndof = inciter::g_inputdeck.get< tag::discr, tag::ndof >();
   std::size_t ncomp = U.nprop()/rdof;
 
   auto beta_lim = 2.0;
 
   for (std::size_t e=0; e<esuel.size()/4; ++e)
   {
-    auto dof_el = ndofel[e];
+    // If an rDG method is set up (P0P1), then, currently we compute the P1
+    // basis functions and solutions by default. This implies that P0P1 is
+    // unsupported in the p-adaptive DG (PDG).
```

Comment:
add "this is a workaround until we have ```rdofel```, which is needed to distinguish between ```ndof```s and ```rdof```s per element for pDG".